### DISCUSSION TITLE
Error When Attempting to Commit Files

### DISCUSSION THREAD

**Discussion by sanogil** [community/community#143845] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,
I’ve been trying to upload files to commit, but after the 'processing files' message, I receive an error saying the commit failed. I’ve attempted this multiple times with different web browsers since yesterday, but I keep encountering the same issue.

I look forward to your response, and thank you for your help.

![commitFailed](https://github.com/user-attachments/assets/c0dc6de3-5393-4917-b3b2-007dde4bfc1d)

**Comment by mastertanzeel** [comment]
The "Commit failed" error in GitHub can happen for several reasons. Here are some troubleshooting steps to help resolve it:

1. Clear Cache and Cookies:
   - Sometimes, old cache or cookies can interfere with actions on websites. Clear your browser cache and cookies, then try committing again.

2. Disable Browser Extensions:
   - Certain browser extensions, especially ad blockers or privacy tools, may interfere with GitHub. Try disabling extensions or using an incognito window to see if that resolves the issue.

3. Check File Size Limits:
   - GitHub has a file size limit of 100 MB per file. If any files are larger, you may need to reduce their size or use [Git Large File Storage (LFS)](https://git-lfs.github.com/) for them.

4. Check Network Issues:
   - Poor or unstable internet connections may cause issues during uploads and commits. Try using a different network if possible or check for connectivity issues.

5. Try Another Browser:
   - You mentioned trying different browsers, which is great. If it’s still not working, try updating your browser to the latest version or use a browser like Chrome, Firefox, or Edge if you haven’t tried them yet.

6. Use GitHub Desktop or Command Line:
   - If the issue persists in the web interface, you can try using [GitHub Desktop](https://desktop.github.com/) or the Git command line to commit and push your files to the repository. This often bypasses issues with the web UI.

7. Check GitHub Status:
   - GitHub occasionally has service interruptions. Check [GitHub's status page](https://www.githubstatus.com/) to make sure there aren’t any ongoing issues affecting commits.

8. Contact GitHub Support:
   - If none of the above methods work, there may be an issue on GitHub's end. You can contact GitHub Support with details of the issue, including the browsers you tried and any recent changes to your environment (e.g., network settings or security software).

These steps should help you identify the cause of the issue or provide a workaround until it's resolved. Let me know if you need help with any specific step!

**Comment by swimmingbrain** [comment]
Hey sanogil! Maybe try clearing your **browser cache and cookies**, as this can sometimes resolve similar issues during commits.

If that doesn't work, can you provide more details about how exactly you're getting the error or any steps you've already tried so far?

**Comment by techy4shri** [comment]
That can happen sometimes, use terminal commands for ease and to avoid such errors.

**Comment by sanogil** [comment, marked as answer]
Thank you for the feedback. The issue was that we were trying to upload a file into other folders, but we had accidentally created an extra folder with the same name. So, when it tried to replace the file, it encountered a folder instead and couldn’t make the commit, but it didn’t give us any error message.
